src/main/kotlin/io/lsdconsulting/lsd/distributed/postgres/repository/InterceptedDocumentPostgresRepository.kt

```diff
@@ -11,8 +11,8 @@ import org.postgresql.util.PSQLException
 import javax.sql.DataSource
 
 
-private const val QUERY_BY_TRACE_IDS_SORTED_BY_CREATED_AT =
-    "select * from intercepted_interactions o where o.trace_id = ANY (?) order by o.created_at"
+private const val QUERY_BY_TRACE_IDS_LIMIT =
```

Comment:
Minor: I'd drop the word `limit` or add `with` so it's `QUERY_BY_TRACE_IDS_WITH_LIMIT`

Reply:
Done.